lib/pure/collections/sequtils.nim

```diff
@@ -25,6 +25,22 @@ import macros
 when not defined(nimhygiene):
   {.pragma: dirty.}
 
+
+macro evalOnce(v, exp: untyped, letAssigneable: static[bool]): untyped =
+  expectKind(v, nnkIdent)
```

Comment:
What does this do different from a single 'let' statement?

Reply:
done